deploy/osd-25821-capa-annotator/10-CronJob-4-17.yaml

```diff
@@ -1,6 +1,7 @@
 apiVersion: batch/v1
 kind: CronJob
 metadata:
+  # Name is left as this as to not orphan the existing CronJob
   name: capa-annotator
```

Comment:
I renamed this file for consistency in the repo, but the CronJob name remains the same as to not orphan it in cluster.